app/billing/rest.py

```diff
@@ -86,3 +93,43 @@ def _transform_billing_for_month(billing_for_month):
         "notification_type": billing_for_month.notification_type,
         "rate": rate
     }
+
+
+@billing_blueprint.route('/free-sms-fragment-limit', methods=["GET"])
+def get_free_sms_fragment_limit(service_id):
+
+    financial_year_start = request.args.get('financial_year_start')
+
+    if financial_year_start is None:
+        results = dao_get_all_free_sms_fragment_limit(service_id)
+
+        if len(results) == 0:
+            raise InvalidRequest('no annual billing information for this service', status_code=404)
+        return jsonify(data=[row.serialize() for row in results]), 200
+    else:
+        result = dao_get_free_sms_fragment_limit_for_year(service_id, financial_year_start)
+        if result is None:
+            raise InvalidRequest('no free-sms-fragment-limit-info for this service and year', status_code=404)
+
+        return jsonify(data=result.serialize()), 200
+
+
+@billing_blueprint.route('/free-sms-fragment-limit', methods=["POST"])
```

Comment:
not from here - the admin app will handle auth. This will only accept requests from notify-admin with the admin's secret key, so we don't need to worry about auth here